src/shared/components/mutationTable/column/PanelColumnFormatter.tsx

```diff
@@ -57,6 +56,27 @@ const getGenePanelIds = (props:PanelColumnFormatterProps) => {
   return [];
 };
 
+const getGenePanelLinks = (props:PanelColumnFormatterProps) => {
```

Comment:
You could make this a React Functional Component and type it as such. Then use it in code via JSX:   <GenePanelLinks ... >